docs/reference/aggregations/bucket/composite-aggregation.asciidoc

```diff
@@ -348,6 +350,34 @@ GET /_search
 \... will sort the composite bucket in descending order when comparing values from the `date_histogram` source
 and in ascending order when comparing values from the `terms` source.
 
+====== Missing bucket
+
+By default documents without a value for a given source are ignored.
+It is possible to include them in the response by setting `ignore_missing` to
+`true` (defaults to `false`):
+
+[source,js]
+--------------------------------------------------
+GET /_search
+{
+    "aggs" : {
+        "my_buckets": {
+            "composite" : {
+                "sources" : [
+                    { "product_name": { "terms" : { "field": "product", "missing_bucket": true } } }
+                ]
+            }
+        }
+     }
+}
+--------------------------------------------------
+// CONSOLE
+
+In the example above the source `product_name` will emit an explicit `null` value
+for documents without a value for the field `product`.
+The `order` specified in the source dictates whether the `null` values should rank
+first (ascending order, `asc`) or last (descending order, `desc`).
+
```

Comment:
I think somewhere in the docs we need to say that the missing option is deprecated and will be removed in favour of this

Reply:
Yes, I plan to add the deprecation in the docs during the backport to 6x since the deprecation is not for master. After the backport to 6x I'll remove the missing option and add a note in the breaking change.